### DISCUSSION TITLE
Ring: snapshot working but no notifications or live view

### DISCUSSION THREAD

**Discussion by snarknesor** [koush/scrypted#483] [Q&A] [answered]
I setup Scrypted yesterday and have pulled my two Ring cameras into Homekit. HK settings are all set to allow notifitications for any activity. Snapshots are refreshed every 10-15 seconds if I have the Home app open but am not able to get live view or receive any notifications. I've tried default settings with no luck and have also played around with basically every setting there is from what I can tell, all with no luck. I have an Aqara camera which is HKSV-compatible out of the box and really like the features so would prefer to get Scrypted up and running instead of defaulting back to Homebridge.

**Comment by koush** [comment, marked as answer]
use troubleshooting guide.
